azext_iot/common/certops.py

```diff
@@ -11,24 +11,30 @@
 import datetime
 from os.path import exists, join
 import base64
+from typing import Dict
 from cryptography import x509
 from cryptography.x509.oid import NameOID
 from cryptography.hazmat.primitives import serialization, hashes
 from cryptography.hazmat.primitives.asymmetric import rsa
 
 
 def create_self_signed_certificate(
-    subject, valid_days, cert_output_dir, cert_only=False
-):
+    subject: str,
+    valid_days: int,
+    cert_output_dir: str,
+    cert_only: bool = False,
+    alt_name: str = None
```

Comment:
alt_name, makes me think of subject alternative names which has a specific meaning with x509 certs. Can we change the name of this param to something more specific to file name?

Reply:
alt_file_name

Reply:
latter please